### PR TITLE
Fix #886: Add Affiliation in Update Profile View

### LINKED ISSUE #886: Add Affiliation in Update Profile View
Currently the user cannot set his affiliation to any organization. We would like to add the Affiliation to Update Profile view.

### PR DESCRIPTION
**Pull request by anirudhvenu**
#886: Add Affiliation in Update Profile View

@RishabhJain2018 Although the changes to the html and controllers have been made, affiliation is not being updated in the database.

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
hey @anirudhvenu I will update you on this soon :)

File: frontend/src/views/web/update-profile.html [hunk 1]
```diff
@@ -32,6 +32,15 @@
                             <p ng-message="maxlength"> Last Name is too long. </p>
                         </div>
                     </div>
+                    <div class="input-field align-left"> 
+                        <input id="affiliation" name="affiliation" type="text" class="text-dark-black dark-autofill w-400" ng-model="updateProfile.user.affiliation" ng-maxlength="30" ng-pattern="/^[A-Za-z]*$/" required focus-if="updateProfile.user.affiliation">
```
Comment (RishabhJain2018) [inline frontend/src/views/web/update-profile.html]: @anirudhvenu I think there is no need for `ng-pattern` in affiliation. Is there any specific reason for using it that I am missing ?

File: frontend/src/views/web/update-profile.html [hunk 2]
```diff
@@ -32,6 +32,15 @@
...
+                        <span class="form-icon form-icon-dark"><i class="fa fa-pencil"></i></span>
+                        <label for="affiliation">Affiliation</label>
+                        <div class="wrn-msg text-highlight" ng-messages="updateProfileForm.affiliation.$error" ng-if="updateProfileForm.affiliation.$touched || updateProfileForm.$submitted">
+                            <p ng-message="required">Affiliation is required</p>
+                            <p ng-message="maxlength"> Affiliation is too long. </p>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/update-profile.html]: Please change `Affiliation is too long.` to `Affiliation is too long`

**Comment by RishabhJain2018** [comment]
@anirudhvenu Any update on this ?

**Comment by aka-jain** [comment]
This seems inactive. @RishabhJain2018 please take a call here.

**Comment by deshraj** [comment]
Closing this due to inactivity.

**Comment by RishabhJain2018** [comment]
@aka-jain @deshraj I am taking this up.

**Pull request closed without merging**